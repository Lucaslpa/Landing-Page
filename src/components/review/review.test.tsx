import { render } from 'utils/test-utils';
import { screen } from '@testing-library/react';
import { Review } from './index';
import Lucas from 'assets/lucas.png';

describe('Review', () => {
  it('Should render review text', () => {
    render(
      <Review
        img={Lucas}
        reviewText="revietext"
        reviewerDescription="review descript"
        reviewerName="reviewer"
      />
    );

    const reviewText = screen.getByText(/revietext/gi);
    expect(reviewText).toBeInTheDocument();
    expect(reviewText).toMatchSnapshot();
  });

  it('Should render reviewer description', () => {
    render(
      <Review
        img={Lucas}
        reviewText="revietext"
        reviewerDescription="reviewer description"
        reviewerName="reviewer"
      />
    );

    const reviewerDescription = screen.getByText(/reviewer description/gi);
    expect(reviewerDescription).toBeInTheDocument();
    expect(reviewerDescription).toMatchSnapshot();
  });

  it('Should render reviewer name', () => {
    render(
      <Review
        img={Lucas}
        reviewText="revietext"
        reviewerDescription="reviewer description"
        reviewerName="reviewer"
      />
    );

    const reviewer = screen.getByText(/reviewer/gi);
    expect(reviewer).toBeInTheDocument();
    expect(reviewer).toMatchSnapshot();
  });
});
