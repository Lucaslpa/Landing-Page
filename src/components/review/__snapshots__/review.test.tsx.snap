// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Should render review text 1`] = `
.c0 {
  width: 100%;
  max-width: 32rem;
  height: auto;
  word-wrap: break-word;
  line-height: 2.2rem;
  font-size: 1.3rem;
  color: #7b7b7b;
}

@media (min-width:768px) {
  .c0 {
    font-size: 1.6rem;
    line-height: 2.4rem;
  }
}

@media (min-width:768px) {

}

<p
  class="c0"
>
  revietext
</p>
`;

exports[`Review Should render reviewer description 1`] = `
@media (min-width:768px) {

}

@media (min-width:768px) {

}

<span>
  reviewer description
</span>
`;

exports[`Review Should render reviewer name 1`] = `
@media (min-width:768px) {

}

@media (min-width:768px) {

}

<span>
  reviewer
</span>
`;
